### ISSUE TITLE
`bun install` in redis monorepo does not work

### ISSUE THREAD

**Issue by Electroid** [oven-sh/bun#7360]
```sh
bun install https://github.com/redis/node-redis#redis@4.6.11
cd redis
bun install
```
```
bun add v1.0.14 (8a679615)
 Resolving dependencies
 Resolved, downloaded and extracted [30]
 Saved lockfile

 installed redis@git+https://github.com/redis/node-redis#d6d2064c72b99d34fc88318f3979177e3c89acd4


 10 packages installed [22.91s]
```
```
bun install v1.0.14 (8a679615)
[26.56ms] migrated lockfile from package-lock.json
 Resolving dependencies

error: package "@redis/test-utils" not found registry.npmjs.org/@redis%2ftest-utils 404
 Resolved, downloaded and extracted [30]
error: @redis/test-utils@* failed to resolve
```

It looks like Bun is attempting to find `@redis/test-utils` in the registry, but it instead exists in the `packages/test-utils` directory.

**Comment by dylan-conway** [comment]
this is happening because the workspace package `@redis/test-utils` doesn't have a version in the package json. Our code needs to be adjusted to choose the workspace package in this situation

**Comment by Electroid** [comment]
This also affects https://github.com/brianc/node-postgres

**Comment by dylan-conway** [comment]
This is fixed as of bun `v1.1.8` with #10913
